packages/react-magma-dom/src/utils/utils.test.jsx

```diff
@@ -59,4 +60,34 @@ describe('Utils', () => {
       expect(normalizedScrollLeft).toEqual(scrollLeft);
     });
   });
+  
+  describe('reactNodeToString', () => {
+    it('converts string to string', () => {
+      const actual = reactNodeToString('test');
+      const expected = 'test';
+
+      expect(reactNodeToString(actual)).toEqual(expected);
```

Comment:
Should this be `expect(actual)` since `actual = reactNodeToString('test')`?

Reply:
@silvalaura 
Done